packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -149,3 +170,8 @@ function sessionFromClient(): string | undefined {
     const crossmintSession = document.cookie.split("; ").find((row) => row.startsWith(SESSION_PREFIX));
     return crossmintSession ? crossmintSession.split("=")[1] : undefined;
 }
+
+function refreshTokenFromClient(): string | undefined {
```

Comment:
I would change this and the upper method to `getRefreshTokenFromCookie` and `getSessionTokenFromCookie`